src/ucp/wireup/address.h

```diff
@@ -34,6 +34,15 @@ enum {
 };
 
 
+enum {
+    UCP_ADDRESS_PACK_FLAG_WORKER_GUID = UCS_BIT(0),
+    UCP_ADDRESS_PACK_FLAG_WORKER_NAME = UCS_BIT(1), /* valid only for debug build */
+    UCP_ADDRESS_PACK_FLAG_DEVICE_ADDR = UCS_BIT(2),
+    UCP_ADDRESS_PACK_FLAG_IFACE_ADDR  = UCS_BIT(3),
+    UCP_ADDRESS_PACK_FLAG_EP_ADDR     = UCS_BIT(4)
+};
```

Comment:
minor: what about add ```UCP_ADDRESS_PACK_FLAG_ALL = -1``` to enable all flags?

Reply:
I don't have objections but usually we don't do that